tests/yam/agama/boot_agama.pm

```diff
@@ -96,7 +96,7 @@ sub run {
     $grub_entry_edition->boot();
 
     return if check_var('AGAMA_GRUB_SELECTION', 'rescue_system');
-    if (get_var('EXTRABOOTPARAMS', '') =~ /systemd.unit=multi-user.target/ || check_var('AGAMA_ALERT_POPUP', 'invalid_profile')) {
+    if (get_var('EXTRABOOTPARAMS', '') =~ /systemd.unit=multi-user.target/ || get_var('INST_ABORT')) {
```

Comment:
Honestly I don't understand this condition, but I also didn't understand it before the change of the variable name. What are we doing here? Distinguishing between graphical and remote backends?

Reply:
In fact, we need continue to run puppeteer test while web UI blocked by the popup so we verify installation started only from serial console.

Reply:
> In fact, we need continue to run puppeteer test while web UI blocked by the popup so we verify installation started only from serial console. 
Besides, Joaquin do not want to add other setting any more, so re-use the existing INST_ABORT.

Reply:
we could use a needle here, this complicate the code, is hard to know why...